tests/integration/api/xai/test_api_xai_validity.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+
+import numpy as np
+import pytest
+import torch
+from mmcls.models import build_classifier
+from mmdet.models import build_detector
+from mpa.det.stage import DetectionStage  # noqa
+from mpa.modules.hooks.recording_forward_hooks import DetSaliencyMapHook, ReciproCAMHook
+from mpa.utils.config_utils import MPAConfig
+
+from otx.algorithms.classification.tasks import ClassificationInferenceTask  # noqa
+from otx.cli.registry import Registry
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+
+torch.manual_seed(0)
+
+templates_cls = Registry("external/model-preparation-algorithm").filter(task_type="CLASSIFICATION").templates
+templates_cls_ids = [template.model_template_id for template in templates_cls]
+
+templates_det = Registry("external/model-preparation-algorithm").filter(task_type="DETECTION").templates
+templates_det_ids = [template.model_template_id for template in templates_det]
```

Comment:
Don't use "external/" directory in feature/otx branch.

Reply:
Thank you for the fix!